upgrade/satellite.py

```diff
@@ -168,12 +167,11 @@ def satellite6_upgrade():
         postup_time = datetime.now().replace(microsecond=0)
         logger.highlight('Time taken for Satellite Upgrade - {}'.format(
             str(postup_time-preup_time)))
-    set_hammer_config()
     # Rebooting the satellite for kernel update if any
     reboot(180)
     host_ssh_availability_check(env.get('satellite_host'))
     # Test the Upgrade is successful
-    hammer('ping')
+    run('hammer ping', warn_only=True)
```

Comment:
Non-blocking comment. No need of warn_only here.

Reply:
resolving it in appropriate PR's. I squashed my PR and has only installation related changes.